quickstarts/microsoft.dataprotection/backup-create-storage-account-enable-protection/main.bicep

```diff
@@ -84,19 +272,28 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2021-06-01' = {
   }
 }
 
-resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
-  name: guid(backupVault.id, roleDefinitionId, storageAccount.id)
+resource storageContainerList 'Microsoft.Storage/storageAccounts/blobServices/containers@2023-01-01' = [
+  for item in containerList: {
+    name: '${storageAccountName}/default/${item}'
```

Comment:
instead of `storageAccountName` here use `storageAccount.name` then you can remove the `dependsOn` property entirely

Reply:
Addressed